bucket/deskpins.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.32",
+    "description": "Make any application always on top.",
+    "homepage": "https://efotinis.neocities.org/deskpins",
+    "license": "MIT",
+    "url": "https://efotinis.neocities.org/downloads/DeskPins-1.32-setup.exe#/dl.7z",
+    "hash": "70bfd44e774837e52bc83f2c128de7d164251e513f6ebea4a70e2073e28ecd2a",
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
+    "bin": "deskpins.exe",
```

Comment:
What are available parameters?

Reply:
Do you mean that `bin` is not needed when there are no command-line parameters available?

Reply:
No... I mean: What are the command line parameters avaialable??

Reply:
I don't think there are any command-line parameters available.
I looked through the product page(https://efotinis.neocities.org/deskpins/index.html) and the manual (`DeskPins.chm` in the binary release), but cannot find any informations about command-line parameters.

Reply:
Then remove it